nuxeo-core/nuxeo-core-io/src/main/resources/OSGI-INF/marshallers-contrib.xml

```diff
@@ -31,7 +31,6 @@
     <register class="org.nuxeo.ecm.core.io.marshallers.json.enrichers.ChildrenJsonEnricher" enable="true" />
     <register class="org.nuxeo.ecm.core.io.marshallers.json.enrichers.HasFolderishChildJsonEnricher" enable="true" />
     <register class="org.nuxeo.ecm.core.io.marshallers.json.enrichers.ContextualParametersJsonEnricher" enable="true" />
-    <register class="org.nuxeo.ecm.core.io.marshallers.json.enrichers.SubtypesJsonEnricher" enable="true" />
```

Comment:
Shouldn't you remove that class as well?
Or if there's will to keep it for backward compat (not a bad idea), it should be marked deprecated.

Reply:
The idea was to leave it in case a client uses or extends it in a custom enricher, so I marked it as deprecated.